ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -72,8 +72,32 @@ <h3 class="title">
         <mat-icon>delete</mat-icon>
         <span>DROP INDEX</span>
       </button>
+      <button
```

Comment:
After dropping and restoring the table , other tables details are visible in the object detail for selected table : https://drive.google.com/file/d/1i_MlrpeXVUYJ80cmspQvyMbxATdrCsRV/view?usp=sharing&resourcekey=0-9VneuLz53HnuXNRw0uW6sA

Reply:
Cause because of synthetic key rendering issue. And it is fixed.